marketplacetests/payment/regions/in_app.py

```diff
@@ -0,0 +1,35 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
From what I can tell, this is an app object for the "Testing In-App-Payments" test app. If that is correct then I don't think it really belongs here under `marketplacetests/payment/regions/` as it is not a part of the payment system.  I suggest putting it elsewhere.